pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/ItemsRule.scala

```diff
@@ -147,4 +155,41 @@ object ItemsRule extends FieldMergeRule with MergerLogging {
         ))
     }
   }
+
+  /**
+    * Sierra records are created from the Sierra / Calm harvest.
+    * We merge the Sierra item ID into the Calm item.
+    *
+    * If an item is digitised via METS, it is linked to the Sierra record.
+    * We merge that into the Calm item.
+    */
+  private val mergeIntoCalmTarget = new PartialRule {
+    val isDefinedForTarget: WorkPredicate = WorkPredicates.calmWork
+    val isDefinedForSource
+      : WorkPredicate = WorkPredicates.metsWork or WorkPredicates.sierraWork
+
+    def rule(target: UnidentifiedWork,
+             sources: NonEmptyList[TransformedBaseWork]): FieldData = {
+
+      // As the Calm transformer always adds 1 item with 1 location, this is safe
```

Comment:
I'm a bit uncomfortable with relying on runtime behaviour in another service - could you add a check for the item number to `WorkPredicates.calmWork` (or make a `singleItemCalm` predicate)? Or do it with `headOption`, whatever you'd prefer but just keeping this safety within the merger.

Reply:
Yea, I copied this from before, but happy to improve.

Reply:
I almost went down the route of trying to type this., but just seemed to add way more complexity. 

> in another service

What do you mean?

Reply:
Say that something changes in the calm transformer (the other service) that means calm works can have > 1 items - that would break the merger. Tests wouldn't necessarily catch it because there would have been no changes in the merger itself.

When we're relying on runtime checks/behaviour rather than types for safety, I think it's preferable to keep the assumptions within one service.

You don't have to do this in a typesafe way if you think it makes it too complex, just do it like the single-item Sierra checks:

```scala
val singleItemCalm: WorkPredicate = satisfiesAll(calmWork, singleItem)
```

Reply:
Added it to the predicate as it felt like the most explanatory way of doing it.

Reply:
This still doesn't help that much as everything will still pass if we forget to look at the merger if we change the transformer. I've just defined the `calmWork` as you have above as there is no other type of calm work.